src/test/java/apoc/math/RegressionTest.java

```diff
@@ -0,0 +1,90 @@
+package apoc.math;
+
+import org.apache.commons.math3.stat.regression.SimpleRegression;
+import org.junit.Rule;
+import org.junit.Test;
+import org.neo4j.driver.v1.*;
+import org.neo4j.harness.junit.Neo4jRule;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author <a href="mailto:ali.arslan@rwth-aachen.de">AliArslan</a>
+ */
+public class RegressionTest {
```

Comment:
@jexp would you please elaborate what is an embedded test in this context. Can you point me toward some example?